### PR TITLE
Add GPU IaaS example with Jupyter.

### PR DESCRIPTION
**Pull request by yohell**

### CODE REVIEW

File: data-science-platform/examples/gpu-sd-iaas-jupyter/index.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+Add to SSH-config (eg `~/.ssh/config`):
+
+```ssh
+Host jupyter-demo
+  HostName [associated IP in Horizon]
+  User ubuntu
+  ProxyJump sshuser@dsp.aida.scilifelab.se
+  ServerAliveInterval 10
+  LocalForward 8888 localhost:8888
```
Comment (eryl) [inline data-science-platform/examples/gpu-sd-iaas-jupyter/index.md:53]: We could add comments about what ports these relate to

Comment (eryl) [reply data-science-platform/examples/gpu-sd-iaas-jupyter/index.md:53]: Nevermind, should have continued reading

File: data-science-platform/examples/gpu-sd-iaas-jupyter/index.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+source .venv/bin/activate
+pip install -r requirements.txt
+```
+
+**Note**:
+The restrictivity of the DSP inspecting http proxy is continually updated, to
+adjust to updates in public repositiories that make them more or less
+appropriate for secure environments. For example, publicly accessible granular
+download counters are increasingly popular despite constituting a trivially
+exploitable data exfiltration method.
```
Comment (eryl) [inline data-science-platform/examples/gpu-sd-iaas-jupyter/index.md:99]: Would it be of public interest to point to an article which explains this data exfiltration? At least for me it was an eye-opener.

**Comment by eryl** [comment]
I think this look good, but will wait for @pontus to chime in

**Review by pontus — APPROVED** [review]

File: data-science-platform/examples/gpu-sd-iaas-jupyter/index.md (hunk 2 above)
Comment (pontus) [reply data-science-platform/examples/gpu-sd-iaas-jupyter/index.md:99]: I don't think it's needed here. For later, I hope we can offer PIs the choice of what to allow with help text and if dare, our evaluation of associated risks. Those should definitely have read more bits (and the same information without the ability to choose should likely be available to users).

**Comment by pontus** [comment]
No problem with merging, see comments for ssh if you want to regard  them.

File: data-science-platform/examples/gpu-sd-iaas-jupyter/index.md (hunk 2 above)
Comment (pontus) [reply data-science-platform/examples/gpu-sd-iaas-jupyter/index.md:99]: Somewhat related; if there are large scale data sources (something people often want to use and would be generally known), we probably want to consider them for access as well.

Comment (yohell) [reply data-science-platform/examples/gpu-sd-iaas-jupyter/index.md:99]: Re: data sources, we can consider adding those on a case by case basis on request.

**Pull request merged**